### PR TITLE
chore(changelog): add entry for `Unstable_Button`

### PR DESCRIPTION
**Pull request by shadoath**
Final part of #375

### CODE REVIEW

File: libs/spark/CHANGELOG.md [hunk 1]
```diff
@@ -82,6 +82,38 @@ _This section details previews of breaking changes or experimental features that
...
+    - `disableElevation`: Removed.
+    - `disableFocusRipple`: Removed.
+    - `disableRipple`: Removed.
+    - `disableTouchRipple`: Removed.
+    - `centerRipple`: Removed.
+    - `focusRipple`: Removed.
+    - `TouchRippleProps`: Removed.
+    - `variant`:
+      - Previous options: `text`, `outlined` and `contained`.
+      - New options: `primary`, `stroked`, `ghost`, and `destructive`.
```
Comment (WilliamKelley) [inline libs/spark/CHANGELOG.md:102]:
```suggestion
      - Removed values: `contained`, `outlined`, `text`.
      - Added values: `primary`, `stroked`, `ghost`, `destructive`.
```
consistent terminology (add / remove); reordering values to emphasize the old-to-new map

File: libs/spark/CHANGELOG.md [hunk 2]
```diff
@@ -82,6 +82,38 @@ _This section details previews of breaking changes or experimental features that
...
+        - Only `destructive` is new, the others were renamed.
+  - Planned Migration from current `Button`:
+    - `color={*}` -> _removed_
+    - `disableElevation={*}` -> _removed_
+    - `disableFocusRipple={*}` -> _removed_
+    - `disableRipple={*}` -> _removed_
+    - `disableTouchRipple={*}` -> _removed_
+    - `centerRipple={*}` -> _removed_
+    - `focusRipple={*}` -> _removed_
```
Comment (WilliamKelley) [inline libs/spark/CHANGELOG.md:111]:
```suggestion
    - `disableElevation` -> _removed_
    - `disableFocusRipple` -> _removed_
    - `disableRipple` -> _removed_
    - `disableTouchRipple` -> _removed_
    - `centerRipple` -> _removed_
    - `focusRipple` -> _removed_
```
These are `boolean` props, so the common usage will be just its presence

File: libs/spark/CHANGELOG.md [hunk 3]
```diff
@@ -82,6 +82,38 @@ _This section details previews of breaking changes or experimental features that
...
+    - `TouchRippleProps={*}` -> _removed_
+    - "variant" prop:
+      - `"contained"` -> `"primary"`
+      - `"text"` -> `"ghost"`
+      - `"outlined"` -> `"stroked"`
```
Comment (WilliamKelley) [inline libs/spark/CHANGELOG.md:116]:
```suggestion
    - `variant="contained"` -> `variant="primary"`
    - `variant="outlined"` -> `variant="stroked"`
    - `variant="text"` -> `variant="ghost"`
```
each line should be as direct as possible; ordering kept the same as in the previous section

**Review by WilliamKelley — CHANGES_REQUESTED** [review]

**Comment by WilliamKelley** [comment]
Can you move the section higher up to be in alphabetical order? So right above `Unstable_CssBaseline` and below the lowercase unstable utilities

**Comment by shadoath** [comment]
> Can you move the section higher up to be in alphabetical order? So right above Unstable_CssBaseline and below the lowercase unstable utilities

Done!

**Review by slingner — APPROVED** [review]

**Pull request merged**